lib/build-time-components/fa-icon.js

```diff
@@ -0,0 +1,80 @@
+/* eslint-env node */
+"use strict";
+
+const { default: BuildTimeComponent, interpolateProperties } = require('ember-ast-helpers/build-time-component');
+const { getAbstractIcon } = require('../fontawesome-helpers');
+const { interpolateSize } = require('./utils');
+
+module.exports = class FaIconComponent extends BuildTimeComponent {
+  constructor(node, opts = {}) {
+    super(node, Object.assign({ tagName: 'svg', ariaHidden: true }, opts));
+    this._syntax = opts.transform.syntax;
+    this.classNames = ['svg-inline--fa'];
```

Comment:
This might get a little dicey as `svg-inline--fa` is configurable. This can be grabbed from `fontawesome.config.replacementClass`

Reply:
Ah, I wasn't aware of that. I will grab that from the config and update this. Thanks.